Sources/Extensions/Foundation/FileManagerExtensions.swift

```diff
@@ -56,5 +56,20 @@ public extension FileManager {
         return nil
     }
 
+    /// SwifterSwift: Decode Encodable object from file.
+    ///
+    /// - Parameters:
+    ///   - type: type of the object that is being decoded.
+    ///   - decoder: JSONDecoder.
+    ///   - url: URL a file with saved object data.
+    /// - Throws: Throws any errors thrown by JSONDecoder.
+    /// - Returns: An object of specified type. Returns nil if failed to read from URL.
+    public func decode<T: Decodable>(as type: T.Type, using decoder: JSONDecoder = JSONDecoder(), from url: URL) throws -> T? {
+        guard let data = FileManager.default.contents(atPath: url.path) else {
+            return nil
+        }
+        let model = try decoder.decode(type, from: data)
+        return model
```

Comment:
Here you can just return the result `return try decoder.decode(type, from: data)` :))